src/styles/_catalog.scss

```diff
@@ -1,22 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, 200px);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  justify-content: center;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
     height: 300px;
+
     border: 1px solid #ccc;
     border-radius: 5px;
+
+    padding: 1em 3em;
+    display: flex;
+    align-items: center;
+
     margin-bottom: 48px;
   }
+}
+
+@media (min-width: 488px) and (max-width: 767px) {
+  .catalog {
+    display: grid;
+    grid-template-columns: repeat(2, 200px);
+    gap: 48px;
```

Comment:
remove `display: grid;` and `gap: 48px;` from media, you have already added these rules to `.catalog`